lib/cli-engine.js

```diff
@@ -694,6 +690,8 @@ CLIEngine.prototype = {
                     descriptor.meta.hashOfConfig = hashOfConfig;
                     descriptor.meta.results = res;
                 }
+            } else {
```

Comment:
Oh, I see now - sorry, didn't see the highlighted line because it was collapsed. No, I actually missed that. Definitely makes sense to put it up there. I'll update the PR :+1: